src/install/install.zig

```diff
@@ -1396,7 +1396,7 @@ pub const PackageInstall = struct {
                 while (try walker.next()) |entry| {
                     if (comptime Environment.isWindows) {
                         switch (entry.kind) {
-                            .directory, .file => {},
+                            .directory, .file, .sym_link => {},
```

Comment:
this went from ignore symlinks to panic on symlink

https://github.com/oven-sh/bun/pull/9922/files#diff-2d4db027e059280840de7dfe46dda1752467e66c1c3014369ed701b026cd06fcR1442